unit_test_framework.h

```diff
@@ -327,10 +327,12 @@ void assert_not_equal(First&& first, Second&& second, int line_number) {
 
 template <typename First, typename Second>
 void assert_sequence_equal(First&& first, Second&& second, int line_number) {
-    auto it1 = std::begin(first);
-    auto it2 = std::begin(second);
-    auto end1 = std::end(first);
-    auto end2 = std::end(second);
+    using std::begin;
```

Comment:
Can you explain this change to me?

Reply:
https://en.cppreference.com/w/cpp/language/adl#Notes

Reply:
So there are other specializations of begin() and end() that need to be considered? I can follow up with you about this some other time.

Reply:
There are a couple mentioned [here](https://en.cppreference.com/w/cpp/iterator/begin#User-defined_overloads), neither of which is important for us. But I could see this showing up in a third-party container library.